### DISCUSSION TITLE
UI Icons

### DISCUSSION THREAD

**Discussion by lawrencecchen** [proselog/proselog#83] [Ideas]
What icons are you using? We can consider using unocss + iconify to standardize:

https://github.com/unocss/unocss/tree/main/packages/preset-icons

**Comment by egoist** [comment]
i just randomly pick what I want from heroicons and icones

unocss + icons sounds like a good idea, what's the best way to use it with next.js?

**Reply by lawrencecchen** [reply]
For non-vite apps, we can use their cli and import the generated css file from _app.tsx

- cli: https://github.com/unocss/unocss/tree/main/packages/cli
- remix example (the only difference is the css import): https://github.com/unocss/unocss/tree/main/examples/remix

**Reply by egoist** [reply]
Done https://egoist.proselog.com/using-unocss-for-icons

**Comment by limichange** [comment]
https://github.com/gregberge/svgr
